src/pages/signup.js

```diff
@@ -199,6 +214,18 @@ const Signup = () => {
           </Stack>
         </Form>
       </Column>
+      <Modal
+        open={open}
+        passiveModal
+        
+        onRequestClose={() => navigate("/login")}
```

Comment:
Here we have to do two things:
1) close the modal 
2) navigate user to login page

To improve this, we can change onRequestClose handler to 

```javascript
onRequestClose={() => {
    setOpen(false);
    navigate("/login");
}}
```